### DISCUSSION TITLE
Per window compact mode

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#844] [Ideas]
### What feature would you like to see?

currently if compact mode is enabled on one window all other window will be enabled. compact mode should be manager by each window separately   

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/415_

**Comment by keifererikson** [comment]
I just discovered that compact mode is not per window. I would love to see this as a per window feature. I run 4 monitors in many different configurations and being able to have some Zen windows in compact, and some in standard layout would be super beneficial for me.

**Comment by DumbledoreCalrissian** [comment]
I second this. This feature would be much appreciated by people with a vertical monitor in their multi-monitor setups.

**Comment by jannisborgers** [comment]
Having compact mode per window makes a lot of sense when you work with multiple monitors in different sizes. A single visual change would also make clearer which window the command has been sent from.

**Comment by joaomendoncaa** [comment]
here's my vouch to also go forward with a per-window compact mode option 

the CLS, and real estate loss from cross-window compact persistence is very painful

**Comment by Y0me** [comment]
Hope this feature will be implemented, this would be particularly useful for multi-screen setups !

**Comment by Gambloide** [comment]
While not part of the release notes of 1.7.1b, the compact-mode toggle via keyboard-shortcut now only toggles it in the current window.
